From: Thomas Hindoe Paaboel Andersen <phomes@gmail.com>
Date: Tue, 24 Jun 2014 23:07:48 +0200
Subject: sysv-generator: rename PidFile to PIDFile

Spotted by Alexey Shabalin

(cherry picked from commit d171ed1c50ba64928b7fb30ee2ae729fdfe0826b)
---
 src/sysv-generator/sysv-generator.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/sysv-generator/sysv-generator.c b/src/sysv-generator/sysv-generator.c
index 18dae5c..9778fbc 100644
--- a/src/sysv-generator/sysv-generator.c
+++ b/src/sysv-generator/sysv-generator.c
@@ -172,7 +172,7 @@ static int generate_unit_file(SysvStub *s) {
                 fprintf(f, "SysVStartPriority=%d\n", s->sysv_start_priority);
 
         if (s->pid_file)
-                fprintf(f, "PidFile=%s\n", s->pid_file);
+                fprintf(f, "PIDFile=%s\n", s->pid_file);
 
         fprintf(f,
                 "ExecStart=%s start\n"
